bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -111,6 +118,81 @@ async def autoreview_status(self, ctx: Context) -> None:
         else:
             await ctx.send("Autoreview is currently disabled.")
 
+    @tasks.loop(hours=72)
+    async def track_forgotten_nominations(self) -> None:
+        """Track active nominations who are more than 2 weeks old."""
+        old_nominations = await self._get_forgotten_nominations()
+        nomination_details = await self._filter_out_tracked_nominations(old_nominations)
+        for nomination, nomination_vote_message in nomination_details:
+            issue_created = await self._track_vote_in_github(nomination)
+            if issue_created:
+                await nomination_vote_message.add_reaction(FLAG_EMOJI)
+
+    async def _get_forgotten_nominations(self) -> List[Nomination]:
+        """Get active nominations that are more than 2 weeks old."""
+        now = arrow.utcnow()
+        nominations = [
+            nomination
+            for nomination in await self.api.get_nominations(active=True)
+            if (now - nomination.inserted_at).days >= OLD_NOMINATIONS_THRESHOLD_IN_DAYS
+        ]
+        return nominations
+
+    async def _filter_out_tracked_nominations(
+            self,
+            nominations: List[Nomination]
+    ) -> List[Tuple[Nomination, discord.Message]]:
+        """Filter the forgotten nominations that are still untracked in GitHub."""
+        untracked_nominations = []
+
+        for nomination in nominations:
+            # We avoid the scenario of this task run & nomination created at the same time
+            if not nomination.thread_id:
+                continue
+
+            thread = await get_or_fetch_channel(nomination.thread_id)
+            if not thread:
+                # Thread was deleted
+                continue
+
+            starter_message = thread.starter_message
+            if not starter_message:
+                # Starter message will be null if it's not cached
+                starter_message = await self.bot.get_channel(Channels.nomination_voting).fetch_message(thread.id)
+
+            if not starter_message:
+                # Starter message deleted
+                continue
+
+            if FLAG_EMOJI in [reaction.emoji for reaction in starter_message.reactions]:
+                # Nomination has been already tracked in GitHub
+                continue
+
+            untracked_nominations.append((nomination, starter_message))
+        return untracked_nominations
+
+    async def _track_vote_in_github(self, nomination: Nomination) -> bool:
+        """
+        Adds an issue in GitHub to track dormant vote.
+
+        Returns True when the issue has been created, False otherwise.
+        """
+        if not GithubAdminRepo.token:
+            log.warn(f"No token for the {GithubAdminRepo.name} repository was provided, skipping issue creation.")
```

Comment:
```suggestion
            log.warning(f"No token for the {GithubAdminRepo.name} repository was provided, skipping issue creation.")
```
warn is a deprecated alias to warning

Reply:
This isn't resolved.